README_SYCL.md

```diff
@@ -37,7 +35,8 @@ Note: Intel FPGAs cannot be targeted together with other Intel hardware. This is
 ### Building for Intel GPUs
 
 * `alpaka_SYCL_ONEAPI_GPU`: set to `ON` to enable compilation for Intel GPUs.
-* `alpaka_SYCL_ONEAPI_GPU_DEVICES`: semicolon-separated list of one or more Intel GPUs to compile for. Check the output of `ocloc compile --help` and look at the possible values for the `-device` argument for valid values to supply here. Default: `bdw`.
+* `alpaka_SYCL_ONEAPI_GPU_DEVICES`: semicolon-separated list of one or more Intel GPUs to compile for. The possible values for the devices are listed in the [UsersManual](https://intel.github.io/llvm-docs/UsersManual.html#generic-options) under the flag `-fsycl-targets`. Default: `intel_gpu_pvc`.
+  Note: currently only one target at a time can be specified (limitation of the Intel Compiler)
```

Comment:
yes, but with the old targets the warp size stuff does not work because the device-specific macros are not defined. That's why we moved to the new targets.

Reply:
From some tests it's not clear if multiple targets are supported or not... still looking into it.

Reply:
OK, after running some more tests, I think the code _is_ compiled for multiple device of the same backend, but _only the first one_ is actually used at runtime.